karma.conf.react.js

```diff
@@ -35,9 +36,22 @@ module.exports = function(config) {
     },
     colors: true,
     concurrency: Infinity,
-    coverageReporter: {
-      dir: 'docs/auth-coverage',
-      type: 'html',
+    coverageIstanbulReporter: {
+      dir: path.resolve('docs/auth-coverage/'),
+      fixWebpackSourcePaths: true,
+      'report-config': {
+        html: {subdir: 'html'},
+      },
+      reports: ['html', 'lcovonly'],
+      thresholds: {
+        emitWarning: false,
+        global: {
```

Comment:
Have we had those thresholds already? If not we should decide in our team if we want to accept those.